doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -104,14 +104,37 @@ apt-get update \
    ...
 ```
 
-2. Download and setup **freemarker.jar** into a directory. The example commands use `/root` to be consistent with the Docker instructions. If you aren't using Docker, you probably want to store the **freemarker.jar** in your home directory.
+2. This build uses the same gcc and g++ compiler levels as OpenJDK, which might be
```

Comment:
@keithc-ca you removed this section from the non-docker instructions in #9371 and I'm not entirely sure why.

Reply:
That change went with the advice to use cmake; which apparently was only partly reverted.